File: python/atm.py
def rupiah(n):
    return "Rp {:,}".format(n).replace(",", ".")

while True:
    nama = input("Masukkan nama: ")
    cek = input("Nama sudah benar? (TRUE/FALSE): ").upper()
    if cek == "TRUE":
        break

while True:
    nim = input("Masukkan NIM angka saja: ")
    if nim.isdigit():
        break

saldo = int(nim)

print("Nama:", nama)
print("Saldo awal:", rupiah(saldo))

while True:
    print("\n===== MENU ATM =====")
    print("1. Cek Saldo")
    print("2. Tarik Tunai")
    print("3. Setor Tunai")
    print("4. Transfer")
    print("5. Keluar")

    p = input("Pilih: ")

    if p == "1":
        print(rupiah(saldo))

    elif p == "2":
        x = int(input("Nominal tarik: "))
        if x <= saldo:
            saldo -= x
        else:
            print("Saldo tidak cukup")

    elif p == "3":
        x = int(input("Nominal setor: "))
        saldo += x

    elif p == "4":
        x = int(input("Nominal transfer: "))
        if x <= saldo:
            rek = input("Rekening tujuan: ")
            print("Transfer ke", rek, "berhasil")
            saldo -= x
        else:
            print("Saldo tidak cukup")

    elif p == "5":
        break

    else:
        print("Pilihan salah")

File: php/atm.php
<?php

function rupiah($n){
    return "Rp " . number_format($n, 0, ',', '.');
}

while(true){
    $nama = readline("Masukkan nama: ");
    $cek = strtoupper(readline("Nama sudah benar? (TRUE/FALSE): "));
    if($cek === "TRUE") break;
}

while(true){
    $nim = readline("Masukkan NIM angka saja: ");
    if(ctype_digit($nim)) break;
}

$saldo = (int)$nim;

echo "Nama: $nama\n";
echo "Saldo awal: " . rupiah($saldo) . "\n";

while(true){
    echo "\n===== MENU ATM =====\n";
    echo "1. Cek Saldo\n";
    echo "2. Tarik Tunai\n";
    echo "3. Setor Tunai\n";
    echo "4. Transfer\n";
    echo "5. Keluar\n";

    $p = readline("Pilih: ");

    switch($p){
        case "1":
            echo rupiah($saldo) . "\n";
            break;

        case "2":
            $x = (int)readline("Nominal tarik: ");
            if($x <= $saldo) $saldo -= $x;
            else echo "Saldo tidak cukup\n";
            break;

        case "3":
            $x = (int)readline("Nominal setor: ");
            $saldo += $x;
            break;

        case "4":
            $x = (int)readline("Nominal transfer: ");
            if($x <= $saldo){
                $rek = readline("Rekening tujuan: ");
                echo "Transfer ke $rek berhasil\n";
                $saldo -= $x;
            } else echo "Saldo tidak cukup\n";
            break;

        case "5":
            exit;

        default:
            echo "Pilihan salah\n";
    }
}
?>